svf/include/Graphs/BasicBlockG.h

```diff
@@ -258,22 +258,23 @@ class BasicBlockGraph: public GenericBasicBlockGraphTy
 {
 private:
     NodeID id{0};
+    const SVFFunction* fun;
 public:
     /// Constructor
-    BasicBlockGraph() {
+    BasicBlockGraph(const SVFFunction* f): fun(f)
+    {
 
     }
 
-    SVFBasicBlock* addBasicBlockToFunction(const std::string& bbname, const SVFFunction* f)
+    SVFBasicBlock* addBasicBlock(const std::string& bbname)
     {
         id++;
-        SVFBasicBlock* bb = new SVFBasicBlock(id, f);
+        SVFBasicBlock* bb = new SVFBasicBlock(id, fun);
```

Comment:
when use C++ new keyword, always remember to delete in the destructor.

Reply:
GenericGraph's destructor will delete every node from the IDToNodeMap.
We don't need to override the destructor.